### PR TITLE
Owners scribes Abb 29

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Owners and scribe of Abb 29

### CODE REVIEW

File: new/PRS13808QerebtaKrestos.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Qǝrǝbta Krǝstos</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="other"/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
+                </person>
+                <listRelation>
+                    <relation name="saws:hasOwned" active="PRS13808QerebtaKrestos" passive="BNFabb29"/>
+                            <relation name="saws:isRelatedTo" active="PRS13808QerebtaKrestos" passive="PRS13807KanaferaKrestos"/>
```
Comment (eu-genia) [inline new/PRS13808QerebtaKrestos.xml:58]: I believe that saws relations are only for works, not for persons

File: new/PRS13807KanaferaKrestos.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Kanāfǝra Krǝstos</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="other"/>
+                    <birth/>
+                    <death/>
+                    <floruit notBefore="1700" notAfter="1830"/>
+                </person>
+                <listRelation>
+                    <relation name="saws:hasOwned" active="PRS13807KanaferaKrestos" passive="BNFabb29"/>
+                    <relation name="saws:isRelatedTo" active="PRS13807KanaferaKrestos" passive="PRS13808QerebtaKrestos"/>
```
Comment (eu-genia) [inline new/PRS13807KanaferaKrestos.xml:58]: I believe that saws relations may not be used for persons

File: new/PRS13809ZamanfasQeddus.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Zamanfas Qǝddus</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="other"/>
+                    <birth/>
+                    <death/>
+                    <floruit notBefore="1700" notAfter="1830"/>
+                </person>
+                <listRelation>
+                    <relation name="saws:hasOwned" active="PRS13809ZamanfasQeddus" passive="BNFabb29"/>
+                    <relation name="saws:isRelatedTo" active="PRS13809ZamanfasQeddus" passive="PRS13808QerebtaKrestos"/>
```
Comment (eu-genia) [inline new/PRS13809ZamanfasQeddus.xml:58]: I believe saws is only for works

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13806Elfyos.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        <revisionDesc>
+            <change who="DN" when="2022-05-02">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">እልፍዮስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎlfyos</persName>
```
Comment (thea-m) [inline new/PRS13806Elfyos.xml:49]: or ʾƎlfǝyos?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/PRS13806Elfyos.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply new/PRS13806Elfyos.xml:49]: I would also assume it is ʾƎlfǝyos, but BM has mostly ʾƎlfyos

File: new/PRS13806Elfyos.xml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+                    <faith type="EOTC"/>
+                    <occupation type="literary"/>
+                    <floruit notBefore="1700" notAfter="1820"/>
+                </person>
+                <listRelation>
+                    <relation name="saws:isCopierOf" active="PRS13806Elfyos" passive="BNFabb29_p3"/>
```
Comment (thea-m) [inline new/PRS13806Elfyos.xml:55]: Sorry, I think the correct way to reference this is BNFabb29#p3?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/PRS13806Elfyos.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply new/PRS13806Elfyos.xml]: Yes, indeed.

**Review by thea-m — APPROVED** [review]

**Comment by thea-m** [comment]
It can be merged once the app is back online. We should not merge while it'S offline (I write it to remind myself :) )

**Comment by DenisNosnitsin1970** [comment]
Could you please merge, if there are no other requests?

**Review by eu-genia — APPROVED** [review]

**Pull request merged**